src/test/java/seedu/address/logic/commands/EditNotesCommandTest.java

```diff
@@ -0,0 +1,82 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_NOTED_INDEX;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_NOTE;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static seedu.address.testutil.TypicalIndexes.INDEX_OUTOFBOUND_NOTE;
+import static seedu.address.testutil.TypicalIndexes.INDEX_OUTOFBOUND_PERSON;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+import static seedu.address.testutil.TypicalNotes.DUPLICATE_NOTE;
+import static seedu.address.testutil.TypicalNotes.TYPICAL_NOTE;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.AddressBook;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.Note;
+import seedu.address.model.person.Person;
+import seedu.address.testutil.PersonBuilder;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * EditCommand.
+ */
+public class EditNotesCommandTest {
+
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void execute_editNotesCommand_success() throws Exception {
+        // Create the person and expected models as in the original
+        Person personToEdit = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
+
+        // Define the edited person with updated notes
+        Person editedPerson = new PersonBuilder(personToEdit).withNotes(
+                TYPICAL_NOTE.getNote(), "Likes dumplings").build();
```

Comment:
Better to have "Likes dumplings" defined as a final static var elsewhere but minor issue.